.github/ISSUE_TEMPLATE/add_dataset.yml

```diff
@@ -0,0 +1,79 @@
+name: Add dataset
+description: Suggest a new Library, Archive or Museum dataset to include in the Hugging Face Hub.
+title: "[Add Dataset]: "
+labels: ["dataset"]
```

Comment:
We have to agree on this step on the workflow: once a person proposes a dataset addition
- does it immediately gets into the project table and another person can start working on adding it immediately?
- or should it be first validated before getting into the project table?

Some kind of basic validations:
- the dataset is not duplicated
- its license certainly allows being added
- it is indeed a LAM-kind dataset
- ...

If we agree on the validation step, then I would propose:
- Create the issue with "dataset candidate" label
- Once validated, update the label with "dataset"

Reply:
I think it makes sense to have a validation step, I will document the things to check there and we can discuss how to manage that when we meet next. 

I will update the tag to "dataset candidate"